### ISSUE TITLE
[Alan] - typos

### ISSUE THREAD

**Issue by lakeen** [w3c/wai-people-use-web-videos#177]
Sequence 3 Visual  - custom-made joytick **should be** custom-made **joystick**
Sequence 5 Audio - But to often the focus indicator,  **should be**  But **too** often the focus indicator,
Sequence 5 Audio - in small widget  **should be** in **a** small widget
Sequence 7 Audio - The most difficult thing are **should be** The most difficult **things** are
